packages/docusaurus-theme-classic/src/theme/BlogPostItem/index.tsx

```diff
@@ -69,6 +71,22 @@ function BlogPostItem(props: Props): JSX.Element {
     frontMatter.author_image_url ||
     frontMatter.authorImageURL;
 
+  // details on structured data support: https://developers.google.com/search/docs/data-types/article#non-amp
+  // and https://schema.org/BlogPosting
+  const blogPostStructuredData = {
+    '@context': 'https://schema.org',
+    '@type': 'BlogPosting',
+    headline: title,
+    description,
+    image: [image],
```

Comment:
Why not `authorImageURL`? And since there may not be image for blog post, then we don't need to add `image` field, right?

Reply:
You're right, the image field may not be populated - maybe it's worth having `authorImageURL` as the fallback if it isn't.  I'll make that change

Reply:
> authorImageURL as the fallback if it isn'

Are you sure that's a good idea for blog post image? Instead this may be better to use only `image` related to blog post?

Reply:
I'm somewhat biased to have an image rather than not, but it's definitely a choice.  I'll make that change.

Reply:
Done